frontend/device/remarkable/device.lua

```diff
@@ -301,7 +335,12 @@ function Remarkable:setEventHandlers(UIManager)
     end
 end
 
-if isRm2 then
+if isRmPaperPro then
```

Comment:
Wouldn't this make more sense below the other one as the elseif? It comes after both chronologically and alphabetically I believe.

Reply:
Will fix it.